### PR TITLE
Add specs for users and posts

### PR DESCRIPTION
**Pull request by Thinus01**
### General requirements

- Made sure that there are no linter errors.
- Made sure that I used correct flow Gitflow.
- Made sure that I documented my work in a professional way.

### Project requirements
- Added RSpec as a gem to my project.
- Created a Request spec file for all my controllers.
- Made sure that for each action I checked:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Thinus01, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- No linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It is not possible to run the tests, please 🙏🏽 fix the issues 👇🏽 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/posts/index'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the correct template' do
```
Comment (BenMukebo) [inline spec/requests/posts_spec.rb:10]: - [ ] As requested by the previous reviouwer, Kindly fix the tests error 👇🏻 

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @Thinus01, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Spec file include tests that ensures correct view template is rendered. ✔️  
- Spec file include tests that ensures the response body includes the correct placeholder text. ✔️  


 ## Required Changes ♻️
_Check the comments under the review._

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Thinus01** [comment]
rorfixed

Screenshot after fixing linters, showing that the code works on my side.

**Review by rotimiazeez — APPROVED** [review]
Hi @Thinus01 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights.✔️
- `index` and `show` actions for User and Post controllers are tested 😊
- RSpec is correctly added to the gem file 👏🏾 
- All tests passed ✔️
- Good PR description/summary 👍🏼
- Descriptive README file 🙂
- No Linter error ✅️ 

# Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**